### PR TITLE
standardize data label according to how Keras standardize input_data

### PR DESCRIPTION
**Pull request by leonardozcm**
This issue occured to me when I found loss was not decreasing, because keras will expand a dimension to ys at the position just behind batchsize, which is 1, but orca skiped this standardizing.
And when it comes to orca, batchsize will be added to the first dimension later, so labels should be standarized as `y = tf.expand_dims(y, axis=0)`.

Reference to tensorflow keras api r1.15 and pr#3260:
- [traning_utils.py](https://github.com/tensorflow/tensorflow/blob/0396c28d6b32177a6f2df3ee459860cf292c395d/tensorflow/python/keras/engine/training_utils.py#L430)
- [training.py](https://github.com/tensorflow/tensorflow/blob/0396c28d6b32177a6f2df3ee459860cf292c395d/tensorflow/python/keras/engine/training.py#L2511)
- [#3260](https://github.com/intel-analytics/analytics-zoo/pull/3260)

### CODE REVIEW

**Comment by jason-dai** [comment]
Is this similar to https://github.com/intel-analytics/analytics-zoo/pull/3260 @yangw1234 @jenniew ?

**Comment by yangw1234** [comment]
> Is this similar to #3260 @yangw1234 @jenniew ?

Yes, this is similar.

File: pyzoo/zoo/orca/learn/tf/estimator.py [hunk 1]
```diff
@@ -619,12 +619,22 @@ def fit(self, data,
         if checkpoint_trigger is not None:
             checkpoint_trigger = Trigger.convert_trigger(checkpoint_trigger)
 
+        if is_tf_data_dataset(data):
+            def standardize_target_data(x, y):
+                if y is not None and len(y.shape) == 0:
```
Comment (yangw1234) [inline pyzoo/zoo/orca/learn/tf/estimator.py:624]: You may need to checkout if `y` is a single tensor. There are cases where `y` has multiple tensors.

Comment (leonardozcm) [reply pyzoo/zoo/orca/learn/tf/estimator.py:624]: ok, I will check this out.

Comment (jason-dai) [reply pyzoo/zoo/orca/learn/tf/estimator.py:624]: If so, which PR should we merge?

Comment (yangw1234) [reply pyzoo/zoo/orca/learn/tf/estimator.py:624]: #3260 only deals with TFNdarrayDataset (backed by a rdd of ndarray). We now find that tf.data.Dataset also needs to add this.

Comment (jenniew) [reply pyzoo/zoo/orca/learn/tf/estimator.py]: yes, this PR is only for tf.data.Dataset. We may need both.

**Comment by leonardozcm** [comment]
tested on jenkins, and ends with different task failed @yangw1234 
http://10.239.47.210:18888/job/ZOO-PR-Validation/4849/
http://10.239.47.210:18888/job/ZOO-PR-Validation/4848/

**Comment by yangw1234** [comment]
LGTM
